pkg/k8shandler/visualization.go

```diff
@@ -397,58 +417,80 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaRoute() error {
 		}
 	}
 
+	if err := clusterRequest.createOrUpdateKibanaSharedConfigMap(); err != nil {
+		return err
+	}
+
+	if err := clusterRequest.createOrUpdateKibanaSharedConfigRBAC(); err != nil {
```

Comment:
You shouldn't need any of this code since the OLM operator is consuming your static resources and creating RBAC for you.

Reply:
The extra RBAC reconciliation is removed now. I kept the cleanup of the old pair of Role/Rolebinding in `openshift-logging`.